packages/eas-cli/package.json

```diff
@@ -9,53 +9,53 @@
   "bugs": "https://github.com/expo/eas-cli/issues",
   "dependencies": {
     "@expo/apple-utils": "0.0.0-alpha.27",
-    "@expo/config": "6.0.11",
-    "@expo/config-plugins": "4.0.11",
-    "@expo/eas-build-job": "0.2.61",
+    "@expo/config": "6.0.16",
+    "@expo/config-plugins": "4.0.16",
+    "@expo/eas-build-job": "0.2.63",
```

Comment:
Remember to run `npm dist-tag add eas-cli-local-build-plugin@0.0.60 latest` after merging

Reply:
AFAIR this does not matter now. It's handled automatically on every release. 
https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/build/local.ts#L9